platform/app/about/page.tsx

```diff
@@ -0,0 +1,67 @@
+                            <defs>
+                                <linearGradient id="paint0_linear_149_5407" x1="51.5411" y1="25.6928" x2="-0.5" y2="25.6928" gradientUnits="userSpaceOnUse">
+                                    <stop stopColor="#10E9A2"/>
+                                    <stop offset="0.505208" stopColor="#429FD6"/>
+                                    <stop offset="1" stopColor="#9749FB"/>
+                                </linearGradient>
+                            </defs>
+                        </svg>
+                    </div>
+                    <h4 className={styles.featureBoxContentHeader}>Responsive Layout</h4>
+                    <p>We haven`t forgotten about responsive layout. With Startup Framework, you can create a website with full mobile support.</p>
+                </div>
+                <div className={styles.featureBoxContent}>
+                    <div className={styles.featureBoxIcon}>
+                        <svg width="65" height="57" viewBox="0 0 65 57" fill="none" xmlns="http://www.w3.org/2000/svg">
+                            <path d="M60.3613 33.7857C60.3613 34.3549 59.8591 34.8571 59.2899 34.8571H5.71847C5.14927 34.8571 4.64704 34.3549 4.64704 33.7857V5.92857C4.64704 5.35937 5.14927 4.85714 5.71847 4.85714H59.2899C59.8591 4.85714 60.3613 5.35937 60.3613 5.92857V33.7857ZM64.647 5.92857C64.647 2.98214 62.2363 0.571424 59.2899 0.571424H5.71847C2.77204 0.571424 0.361328 2.98214 0.361328 5.92857V42.3571C0.361328 45.3036 2.77204 47.7143 5.71847 47.7143H23.9328C23.9328 50.6272 21.7899 52.904 21.7899 54.1429C21.7899 55.3147 22.7609 56.2857 23.9328 56.2857H41.0756C42.2475 56.2857 43.2185 55.3147 43.2185 54.1429C43.2185 52.971 41.0756 50.5603 41.0756 47.7143H59.2899C62.2363 47.7143 64.647 45.3036 64.647 42.3571V5.92857Z" fill="url(#paint0_linear_149_5403)"/>
```

Comment:
Why we have path here? If u put this to show image, upload the image into public file then give the path of it. There is an example NexBridge/index.tsx